### PR TITLE
Fix 2023 schedule, speakers and sponsors

### PR DESCRIPTION
**Pull request by balinterdi**
tl;dr: The content on https://emberfest.eu/2023 is a copy of this year's and this PR fixes that.

Here is the longer story (kinda important so that we don't make the same mistake next year):

Tomek pointed to me that the 2023 schedule is a copy of this year's schedule.

I remember checking the 2023 schedule myself *before* posting this year's one and I remember it was correct back then. To my surprise, Tomek was right: the 2023 schedule is now the same as this year's which this PR fixes.

I'm fairly certain what happened is that when we archived last year's content, we didn't make a full copy of the content (stored in `.yml` files), so the 2023 pages just kept referring to `data.site.schedule`. Which was fine until the schedule wasn't update for this year 😅

### CODE REVIEW

**Review by jkintscher — APPROVED** [review]
If it’s fixed, it’s fixed! Thank you (and Tomek)!
